### ISSUE TITLE
Update READMEs and package info in components

### ISSUE THREAD

**Issue by froddd** [coopdigital/coop-frontend#42]
Newly added components need to have their READMEs checked to make sure there isn't anything specific to older repository. 

The github repository stated in the `package.json` also needs updating, as well as any other snippet of the package info, as required.

**Comment by matt-tyas** [comment]
Yeah. I've done some of them, but there is a fair bit of house keeping still required.

**Comment by omidantilong** [comment]
I'm going to pick this up and do a big sweep on all the docs as soon as possible. Some stuff is in there, just hidden away.
